docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md

```diff
@@ -152,11 +150,11 @@ For example, if your goal is to allow the platform software to manage YugabyteDB
 
 ```sh
 curl -s https://raw.githubusercontent.com/yugabyte/charts/master/rbac/platform-namespaced.yaml \
-  | sed "s/namespace: <SA_NAMESPACE>/namespace: <namespace>"/g \
+  | sed "s/namespace: <serviceaccount_namespace>/namespace: <namespace>"/g \
```

Comment:
`platform-namespaced.yaml` has a placeholder `<SA_NAMESPACE>`. So this updated sed command does not replace the placeholder anymore.

Either
1. Change the command to use the correct placeholder
2. Change the `platform-namespaced.yaml` to use the correct placeholder

Reply:
Changed back to <SA_NAMESPACE>. 
The change will be delivered via a new PR.